src/org/zaproxy/zap/extension/scripts/files/scripts/scripts/extender/Copy as curl command menu.js

```diff
@@ -0,0 +1,76 @@
+// Extender scripts allow you to add completely new functionality to ZAP.
+// The install function is called when the script is enabled and the uninstall function when it is disabled.
+// Any functionality added in the install function should be removed in the uninstall method.
+// See the other templates for examples on how to do add different functionality. 
+
+// The following handles differences in printing between Java 7's Rhino JS engine
+// and Java 8's Nashorn JS engine
+if (typeof println == 'undefined') this.println = print;
```

Comment:
This could be dropped? (The script requires Java 8)
Same comment for other scripts.